VirtoCommerce.Storefront/Domain/Cart/CartConverter.cs

```diff
@@ -496,6 +496,11 @@ public static ShoppingCart ToShoppingCart(this cartDto.ShoppingCart cartDto, Cur
             result.VolumetricWeight = (decimal)(cartDto.VolumetricWeight ?? 0);
             result.Weight = (decimal)(cartDto.Weight ?? 0);
 
+            if (cartDto.ConfiguredGroups != null)
```

Comment:
cartDto.ConfiguredGroups.IsNullOrEmpty

Reply:
It's doing same way as below and above. And I think we already discussed about that